src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -53,8 +54,6 @@ public function __construct(array $mockedNamespaces = array())
             Blacklist::$blacklistedClassNames['\Symfony\Bridge\PhpUnit\Legacy\SymfonyTestsListenerTrait'] = 2;
         }
 
-        $enableDebugClassLoader = class_exists('Symfony\Component\Debug\DebugClassLoader');
```

Comment:
this should be kept here:
`$enableDebugClassLoader = class_exists(DebugClassLoader::class) || class_exists(LegacyDebugClassLoader::class);`